cityofaustin/atd-moped
# Support timeline templates for multiple workgroups

## Associated issues

Closes https://github.com/cityofaustin/atd-data-tech/issues/10238

## Testing
**URL to test:** 
https://deploy-preview-796--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
1. Test project creation, I removed the option to choose a type from the project workflow. No milestones should show up after a project is created
2. Navigate to the timeline tab. Open the button dropdown, select "From Template"
3. A modal should appear, with a disabled button and an autocomplete to choose templates. 
4. after choosing the AMD choice, the list should populate with preselected milestones. (PMD is still empty because we dont have the "template" listed yet) Toggling them off will not include them in the mutation. 
5. any existing milestone on the table should not show up in the modal. You can test by adding some of the milestones and then opening the modal again and seeing the list is shorter. 

---
#### Ship list
